azure/create-spn.sh

```diff
@@ -19,6 +20,7 @@ show_help()
   echo "Options:"
   echo "  -d, --display-name: The name of Azure AD App."
   echo "  -s, --subscription: The Azure Subscription ID used by EDB Cloud."
+  echo "  -y, --years:        The Number of years for which the credentials will be valid. Default: 1 year."
```

Comment:
```
ERROR: argument --years: invalid int value: '1.2'
```

I am thinking if that can be better if we mention this `-y` only accept integer typed input?

Reply:
Actually I cannot find any rules about this argument from Azure..but it's good to add some words about this.